DevelopmentTools/officialMap/HOCalibTest.txt

```diff
@@ -0,0 +1,84 @@
+#The valid run modes are: HB, HE, HF, HO
+#The run mode we have right now is: HO
+#The valid run modes are: LMap, EMap, FMap, HT, LMapCalib, EMapCalib
+#The produce mode we have right now is: LMapCalib
+#Constructing HO Calib Object...
+#Dumping HB Calib LMap Object...
+#  side   eta   phi  dphi depth   rbx   sector    rm rm_fi rm_ch   det cpCol cpRow  cpLC crate   HTR  fpgahtr_fiber
```

Comment:
rename, in dumper.cc

Reply:
# Side Eta Phi dPhi CH_TYPE Det RBX Wedge QIE8 QIECH RM RM_FI FI_CH Trunk cpCol cpRow cpCpl cpLC cpOct ppCol ppRow ppCpl ppLC dodec Crate uHTR uHTR_FI FEDid QIE8id